plugins/modules/copr.py

```diff
@@ -77,6 +77,13 @@
   community.general.copr:
     state: absent
     name: '@copr/integration_tests'
+
+- name: Install Caddy
+  community.general.copr:
+    name: '@caddy/caddy'
+    chroot: fedora-rawhide-{{ ansible_architecture }}
```

Comment:
Since using top-level facts will get deprecated soon, let's change it already now:
```suggestion
    chroot: fedora-rawhide-{{ ansible_facts.architecture }}
```

Reply:
Thanks for this info (I just started using Ansible). I've updated several answers on StackOverflow as well so that others don't make this mistake.